modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/GridCacheDatabaseSharedManager.java

```diff
@@ -3331,6 +3348,14 @@ private void waitCompleted() throws IgniteCheckedException {
         /** */
         private long lastCpTs;
 
+        /** Long JVM pause threshold. */
+        private final int longJvmPauseThreshold =
+            getInteger(IGNITE_JVM_PAUSE_DETECTOR_THRESHOLD, DEFAULT_JVM_PAUSE_DETECTOR_THRESHOLD);
+
+        /** Long JVM puase detector enabled flag. */
+        private final boolean pauseDetectorEnabled =
```

Comment:
why can`t u use this static fields from LongJVMPauseDetector class ? change access and use it, isn`t it ?